app/tracker/page.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import React, { useState, useEffect } from "react";
+import "./Tracker.css";
+import { ShoppingCart, Settings, Package, Check } from "lucide-react";
+
+
+const generateTrackingNumber = () => {
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+  return Array.from({ length: 12 }, () => characters.charAt(Math.floor(Math.random() * characters.length))).join('');
+};
```

Comment:
transfer to another file lang, maybe put it in

`/lib/utils`,

also pa convert to regular named functions instead of arrow functions